core/src/types/params.rs

```diff
@@ -43,6 +38,16 @@ impl Params {
 	}
 }
 
+impl Into<Value> for Params {
```

Comment:
Should implement `From<Params> for Value` instead as per rust guideliness if possible.